USAGE.md

```diff
@@ -1,370 +1,707 @@
+# The ATT&CK Data Model
 
-## Properties
-The following is a table mapping of ATT&CK properties, the old ATT&CK MediaWiki names, and the new STIX properties. Some of these properties are standard STIX properties, while others were custom-created for compatibility with ATT&CK. These properties are accessed from STIX objects as JSON properties.
+The data in this repository is STIX2.0 and divided into three folders, one for each domain of ATT&CK. These domains generally follow the same format with a few departures. Domain differences will be noted in the relevant sections of this document. 
 
-### Migrating from MediaWiki
+STIX uses a different terminology than ATT&CK. The following table is a mapping of ATT&CK concepts to STIX 2.0 objects:
 
-### Common properties (on all objects)
-ATT&CK Property | ATT&CK MediaWiki | STIX Properties
---------------- | ---------------- | ---------------
-**Entry ID**    | `Has ID` | `external_references[i].external_id` where `external_references[i].source_name == "mitre-attack"`
-**Entry URL**   | `URL` | `external_references[i].url` where `external_references[i].source_name == "mitre-attack"`
-**Entry Title** | `Has display name` | `name`
-**Entry Text**  | `Has description` | `description`
-**Citation**    | `Citation reference` | `external_references`
-**Deprecated**  | `Deprecated` | `x_mitre_deprecated`
-**Revoked**     | `Not available via MediaWiki API` | `revoked`
-**Old ATT&CK ID** | `Not available via MediaWiki API` | `x_mitre_old_attack_id`
+| ATT&CK concept | STIX object type |
+|:------------|:----------|
+| [Matrix](#matrices)              | `x-mitre-matrix` |
+| [Tactic](#tactics)               | `x-mitre-tactic` |
+| [Technique](#techniques)         | `attack-pattern` |
+| [Sub-technique](#sub-techniques) | `attack-pattern` where `x_mitre_is_subtechnique = true` |
+| [Procedure](#procedures)         | `relationship` where `relationship_type = "uses"` and `target_ref` is an `attack-pattern` |
+| [Mitigation](#mitigations)       | `course-of-action` |
+| [Group](#groups)                 | `intrusion-set`  |
+| [Software](#software)            | `tool` or `malware` |
 
+Two additional object types are found in the ATT&CK catalog:
+ 
+| STIX object type | About |
+|:-----------------|:------|
+| `identity`       | Referenced in the `created_by_ref` of all objects to state that the MITRE Corporation created the object |
+| `marking-definition` | Referenced in the `object_marking_refs` of all objects to express the MITRE Corporation copyright |
+
+## Common departures from the STIX spec
+
+There are three general ways that ATT&CK departs from the stock STIX 2.0 format:
```

Comment:
The spec refers to "extending" STIX. Since we're still following the spec, we could say that ATT&CK "extends the predefined" STIX 2.0 format

Reply:
Agreed. I also changed the section title to "Extensions of the STIX Spec" to again move away from the word departures.